### PR TITLE
Upgrade ElasticSearch/Kibana to 7.6.1

### LINKED ISSUE #1805: Update Elasticsearch to 7.6.1
Elasticsearch 7.6.1 available in Opendistro.

### PR DESCRIPTION
**Pull request by vincenzodnp**
Upgraded opendistro_security plugin to 1.6.0
Solves #1571: cluster.routing.allocation.disk.threshold_enabled

# Checklist
- [x] Affected Issues have been mentioned in the Closing issues section
- [x] Documentation has been written/updated
- [x] PR title is ready for changelog and subsystem label(s) applied

Upgrade ElasticSearch and Kibana to 7.6.1 (latest supported by Opendistro).  
The PR fixes a misconfiguration of Elasticsearch inherited by Opendistro preventing the cluster to load balance indexes all over the nodes of the cluster. 

# Closing issues
closes #1805

### CODE REVIEW

File: docs/using_lagoon/index.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Some Docker images and containers need additional customizations from the provid
 
 | Type | Versions | Dockerfile |
 | :--- | :--- | :--- |
-| [Elasticsearch](docker_images/elasticsearch.md) | 6.8, 7.1, 7.3 | [elasticsearch/Dockerfiles](https://github.com/amazeeio/lagoon/tree/master/images/elasticsearch) |
+| [Elasticsearch](docker_images/elasticsearch.md) | 6.8, 7.1, 7.6 | [elasticsearch/Dockerfiles](https://github.com/amazeeio/lagoon/tree/master/images/elasticsearch) |
```
Comment (Schnitzel) [inline docs/using_lagoon/index.md]: let's remove the `7.1` ELK images with this PR as we don't need them anymore

File: docker-compose.yaml [hunk 2]
```diff
@@ -475,7 +475,7 @@ services:
     image: ${IMAGE_REPO:-lagoon}/logs-db
     user: '111111111'
     environment:
-      KEYCLOAK_URL: http://docker.for.mac.localhost:8088
+      KEYCLOAK_URL: http://keycloak:8080
```
Comment (Schnitzel) [inline docker-compose.yaml]: can you explain that change?

File: docker-compose.yaml [hunk 3]
```diff
@@ -499,7 +499,7 @@ services:
     ports:
       - '5601:5601'
     environment:
-      KEYCLOAK_URL: http://docker.for.mac.localhost:8088
+      KEYCLOAK_URL: http://${KEYCLOAK_URL:-docker.for.mac.localhost:8088}
```
Comment (Schnitzel) [inline docker-compose.yaml]: can you explain that change?

**Review by Schnitzel — CHANGES_REQUESTED** [review]
nice, good work, a few small adaptions

File: docs/using_lagoon/index.md (hunk 1 above)
Comment (vincenzodnp) [reply docs/using_lagoon/index.md]: Remove by latest commit

File: docker-compose.yaml (hunk 2 above)
Comment (vincenzodnp) [reply docker-compose.yaml]: it comes from #1526 .
GNU/Linux doesn't know `docker.for.mac.localhost` and both `logs-db` and `logs-db-ui` dont' start properly.
To test lagoon on my local, I had to "merge" the changes @smlx made in the PR mentioned.

File: docker-compose.yaml (hunk 3 above)
Comment (vincenzodnp) [reply docker-compose.yaml]: it comes from #1526 .
GNU/Linux doesn't know `docker.for.mac.localhost` and both `logs-db` and `logs-db-ui` dont' start properly.
To test lagoon on my local, I had to "merge" the changes @smlx made in the PR mentioned.

File: docker-compose.yaml (hunk 2 above)
Comment (Schnitzel) [reply docker-compose.yaml]: okay, but let's remove it from this PR please, we want to keep them separate

File: docker-compose.yaml (hunk 3 above)
Comment (Schnitzel) [reply docker-compose.yaml]: okay, but let's remove it from this PR please, we want to keep them separate

**Review by Schnitzel — CHANGES_REQUESTED** [review]
please remove any commits from other PRs

File: docker-compose.yaml (hunk 2 above)
Comment (vincenzodnp) [reply docker-compose.yaml]: Removed

File: docker-compose.yaml (hunk 3 above)
Comment (vincenzodnp) [reply docker-compose.yaml]: Removed

**Pull request merged**